k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -100,7 +105,90 @@ private void initView(Context context) {
         adapter = new RecipientAdapter(context);
         setAdapter(adapter);
 
-        setLongClickable(true);
+        addDragAndDropFunctionality(context);
+    }
+
+    public void setDragListener(DragListener listener) {
+        dragListener = listener;
+    }
+
+    private void addDragAndDropFunctionality(Context context) {
```

Comment:
Do you mean a reference to the RecipientSelectView ? There is very little interaction between RecipientMvpView and RecipientSelectView for drag and drop (only one call to onClickRecipientExpander()).